src/bsp/platform/cc3220/xi_bsp_time_cc3220_sntp.c

```diff
@@ -4,6 +4,8 @@
  * it is licensed under the BSD 3-Clause license.
  */
 
+#include <socket.h>
```

Comment:
is this needed? since I don't see change in the file. Did it fail to compile before?